tesseract/tesseract_geometry/include/tesseract_geometry/mesh_parser.h

```diff
@@ -203,7 +206,96 @@ inline std::vector<std::shared_ptr<T>> createMeshFromPath(const std::string& pat
     importer.ApplyPostProcessing(aiProcess_OptimizeGraph);
   }
 
-  return createMeshFromAsset<T>(scene, scale, path);
+  return createMeshFromAsset<T>(scene, scale, nullptr);
+}
+
+/**
+ * @brief Create a mesh using assimp from resource
+ * @param resource The located resource
+ * @param scale Perform an axis scaling
+ * @param trianglulate If true the mesh will be trianglulated. This should be done for visual meshes.
+ *        In the case of collision meshes do not triangulate convex hull meshes.
+ * @param flatten If true all meshes will be condensed into a single mesh. This should only be used for visual meshes,
+ * do not flatten collision meshes.
+ * @return
+ */
+
```

Comment:
not sure if the extra space will cause an issue when generating doxygen documentation.